bandit/cli/main.py

```diff
@@ -246,7 +246,7 @@ def main():
                            '(only JSON-formatted files are accepted)'
     )
     parser.add_argument(
-        '--ini', dest='ini_path', action='store', default=None,
+        '--ini', dest='ini_path', action='store', default='.bandit',
```

Comment:
This limits the default to just a single possibility.

Other popular tools, such as [flake8](https://gitlab.com/pycqa/flake8/blob/master/src/flake8/options/config.py#L45), [behave](https://github.com/behave/behave/blob/master/behave/configuration.py#L443-L448), etc., allow the configuration to be discovered from various popular configuration files (to save the project home directory from being polluted with numerous configuration files).

I tend to place all configuration into a `tox.ini` file, for example. This makes sense as Tox is my main driver and single point of entry for running all tests and check on Python projects.